boring/src/sizing/code generator/outputfile.txt

```diff
@@ -0,0 +1,25 @@
+self.add_input('alpha')
```

Comment:
it might be best to .gitignore these output .txt files, tracking them would be duplicate since they also exist in the component